### PR TITLE
feat: adds rest api endpoints

### PR DESCRIPTION
**Pull request by madhavilosetty-intel**
## PR Checklist

- [ ] Unit Tests have been added for new changes
- [ ] API tests have been updated if applicable
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with Apache 2.0 and clearly outlined the added dependency.

### CODE REVIEW

File: packages.config [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Bcl.AsyncInterfaces" version="1.1.0" targetFramework="net48" />
+  <package id="Microsoft.IdentityModel.Abstractions" version="7.2.0" targetFramework="net48" />
+  <package id="Microsoft.IdentityModel.JsonWebTokens" version="7.2.0" targetFramework="net48" />
+  <package id="Microsoft.IdentityModel.Logging" version="7.2.0" targetFramework="net48" />
+  <package id="Microsoft.IdentityModel.Tokens" version="7.2.0" targetFramework="net48" />
+  <package id="Newtonsoft.Json" version="13.0.3" targetFramework="net48" />
```
Comment (rsdmike) [inline packages.config:8]: We already have System.text.JSON -- we should only use one package and migrate off of newtonsoft ( https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/migrate-from-newtonsoft?pivots=dotnet-9-0)

File: src/ProfileRequest.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OpenAMTEnterpriseAssistant
+{
+    public class ProfileRequest : RequestModel
+    {
+        public object cert { get; set; }
+        public object certid { get; set; }
```
Comment (rsdmike) [inline src/ProfileRequest.cs:12]: why object? what about dynamic?

File: packages.config (hunk 1 above)
Comment (madhavilosetty-intel) [reply packages.config]: done

File: src/ProfileRequest.cs (hunk 2 above)
Comment (madhavilosetty-intel) [reply src/ProfileRequest.cs]: done

**Review by matt-primrose — APPROVED** [review]

**Pull request merged**